packages/metamaps/pages/[author_address]/map/[id].tsx

```diff
@@ -78,6 +78,8 @@ export const MapPageComponent: FC<MapPageProps> = ({ dispatch }) => {
     data: items,
   });
 
+  dispatch({ type: 'POPOVER_NAME_INPUT', value: data?.Map_by_pk.name });
```

Comment:
People working on this project should learn about render mechanisms of React and how works hooks.

The content of this method, from line 18 to line 88, is executed at each render. 
It means that:
- the dispatches will be sent many times
- the items will be computed many times

You must wrap this logic into respectively useEffect and useMemo